README.md

```diff
@@ -7,6 +7,12 @@ Management Server component of uProxy for Orgs (UfO)
 [![Travis Status](https://travis-ci.org/uProxy/ufo-management-server-flask.svg)](https://travis-ci.org/uProxy/ufo-management-server-flask)
 [![Code Health](https://landscape.io/github/uProxy/ufo-management-server-flask/master/landscape.svg?style=flat)](https://landscape.io/github/uProxy/ufo-management-server-flask/master)
 
+## Feedback
+
+If you would like to report a problem or request a new feature with the management server or uProxy overall, feel free to use the issue tracker on [this repo](https://github.com/uProxy/ufo-management-server-flask/issues) or on the [uProxy issue tracker](https://github.com/uProxy/uproxy/issues). Wherever you file, please include as much pertinent information as possible, such as the version of the management server being used, date and time the issue was encountered, steps to reproduce the issue, what browser and platform the issue was encountered upon, etc.
```

Comment:
Instead of suggesting to the user some potential info to provide, we should tell them definitively what we need, and provide a format for them to do so.  Does github provide a template for issues?

Manager Server Version:

Browser Type and Version:

Steps to Reproduce:

Applicable Logs:

Applicable Screenshot: